front/pages/api/v1/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -285,9 +290,12 @@ async function handler(
         "App run creation"
       );
 
-      const runRes = await coreAPI.createRunStream({
+      const groups = await GroupResource.listWorkspaceGroupsFromKey(
```

Comment:
As discussed on slack this should be the keyAuth groups which, once we implement X-Dust-User-Id, will correctly represent the calling user's groups

Reply:
If by "keyAuth groups` you mean the groups associated with the key, it's already the case.